recipe/meta.yaml

```diff
@@ -31,12 +31,13 @@ test:
     - xmlwf -h
 
 about:
-  home: http://expat.sourceforge.net/
+  home: https://expat.sourceforge.net/
   license: MIT
   license_family: MIT
```

Comment:
I believe `license_family` should be removed. At best it seems to add no value, at its worst it's confusing. This decision is yours to make.

Reply:
Normally we try to keep a license family in all of the recipes for the sake of consistency.

Reply:
Yes, it is used in a product. We should continue to update it.